includes/plugin-instrumentation.php

```diff
@@ -65,19 +106,25 @@ public function rzpTrackDataLake($properties)
         }
     }
 
-    public function getDefaultProperties()
+    public function getDefaultProperties($timestamp = true)
```

Comment:
@abdulwahidsharief timestamp is always required, it's not optional.

Reply:
@enishant in datalake we have timestamp as required field in schema it will be data redundancy if we keep same timestamp in properties